From 898d30113adc366854dbffd402962e9bcc6337ae Mon Sep 17 00:00:00 2001
From: Patrick Hemmer <phemmer@chewy.com>
Date: Tue, 29 Nov 2016 13:56:44 -0500
Subject: [PATCH] Low: libservices: treat systemd service reloading as OK

Previously if pacemaker performed a monitor operation while a service was
reloading, pacemaker treated it as if the service were stopped/failed.
---
 lib/services/systemd.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/lib/services/systemd.c b/lib/services/systemd.c
index e6e1114..ea01365 100644
--- a/lib/services/systemd.c
+++ b/lib/services/systemd.c
@@ -552,6 +552,8 @@ systemd_unit_check(const char *name, const char *state, void *userdata)
 
     } else if (g_strcmp0(state, "active") == 0) {
         op->rc = PCMK_OCF_OK;
+    } else if (g_strcmp0(state, "reloading") == 0) {
+        op->rc = PCMK_OCF_OK;
     } else if (g_strcmp0(state, "activating") == 0) {
         op->rc = PCMK_OCF_PENDING;
     } else if (g_strcmp0(state, "deactivating") == 0) {
-- 
1.8.3.1

